suite-native/module-add-accounts/src/components/SelectableAccountTypeItem.tsx

```diff
@@ -0,0 +1,113 @@
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+}));
+
+const radioWrapper = prepareNativeStyle(_ => ({
+    width: '100%',
+    alignItems: 'flex-end',
+    paddingTop: 12,
+}));
+
+const subtitleWrapper = prepareNativeStyle(_ => ({
+    paddingBottom: 4,
+}));
+
+const badgeWrapper = prepareNativeStyle(_ => ({
+    paddingTop: 4,
+}));
+
+type SelectableAccountTypeItemProps = {
+    accountType: AccountType;
+    isSelected: boolean;
+    isDefault: boolean;
+    onPress?: (accountType: AccountType) => void;
+};
+
+export const SelectableAccountTypeItem = ({
+    accountType,
+    isSelected,
+    isDefault,
+    onPress,
+}: SelectableAccountTypeItemProps) => {
+    const { applyStyle, utils } = useNativeStyles();
+    const { translate } = useTranslate();
+
+    const handlePress = () => onPress?.(accountType);
+
+    const { titleKey, subtitleKey, descKey } = getIntlKeysForAccountType(accountType);
+
+    return (
+        <TouchableOpacity onPress={handlePress} activeOpacity={0.6}>
+            <Card style={applyStyle(cardStyle, { isSelected })}>
+                <VStack spacing={utils.spacings.extraSmall}>
+                    <Box style={applyStyle(titleWrapper)}>
+                        <Text variant="titleSmall" color="textDefault">
+                            <Translation id={titleKey} />
+                        </Text>
+                        {isDefault && (
+                            <View style={applyStyle(badgeWrapper)}>
+                                <Badge
+                                    key="defaultType"
+                                    variant="green"
+                                    label={translate(
+                                        'moduleAddAccounts.selectAccountTypeScreen.defaultLabel',
+                                    )}
+                                    icon="checkCircle"
+                                />
+                            </View>
+                        )}
+                    </Box>
+                    <Text variant="hint" color="textDefault" style={applyStyle(subtitleWrapper)}>
+                        <Translation id={subtitleKey} />
+                    </Text>
+                    <Text>
+                        <Translation
+                            id={descKey}
+                            values={{
+                                li: chunks =>
+                                    chunks.map(row => (
+                                        <Box flexDirection="row">
+                                            <Text variant="hint" color="textSubdued">
+                                                {'  '}•{'  '}
+                                            </Text>
+                                            <Text variant="hint" color="textSubdued">
+                                                {row}
+                                            </Text>
+                                        </Box>
```

Comment:
Check out `ListItem` component in `suite-native/atoms`. Seems like it could be used here.